### ISSUE TITLE
mkmr failing to cache if XDG_CACHE_HOME isn't set

### ISSUE THREAD

**Issue by Cogitri** [maxice8/mkmr#56]
It fails with:

```
Traceback (most recent call last):
  File "/usr/lib/python3.8/pathlib.py", line 1279, in mkdir
    self._accessor.mkdir(self, mode)
FileNotFoundError: [Errno 2] No such file or directory: '/Cogitri/aports'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/rasmus/.local/bin/mkmr", line 4, in <module>
    main()
  File "/home/rasmus/.local/lib/python3.8/site-packages/mkmr/mkmr.py", line 306, in main
    origin.projectid(token=gl.private_token), retry_transient_errors=True, lazy=True
  File "/home/rasmus/.local/lib/python3.8/site-packages/mkmr/api.py", line 64, in projectid
    cachedir = create_dir(
  File "/home/rasmus/.local/lib/python3.8/site-packages/mkmr/utils.py", line 34, in create_dir
    path.mkdir(mode=0o700, parents=True, exist_ok=True)
  File "/usr/lib/python3.8/pathlib.py", line 1283, in mkdir
    self.parent.mkdir(parents=True, exist_ok=True)
  File "/usr/lib/python3.8/pathlib.py", line 1279, in mkdir
    self._accessor.mkdir(self, mode)
PermissionError: [Errno 13] Permission denied: '/Cogitri'
```

**Comment by maxice8** [comment]
I see

**Comment by maxice8** [comment]
fixed in 0.0.13
